{{cookiecutter.project_slug}}/src/{{cookiecutter.project_module}}/settings.py

```diff
@@ -75,6 +75,7 @@ class Testing(Default):
     """Testing environment configuration."""
 
     def __init__(self, **kwargs):
+        """Initialize the testing environment configuration."""
```

Comment:
We're not being very consistent here. `Testing.__init__` accepts `**kwargs` whereas `Production.__init__` does not.